packages/analytics/metric-cards/src/components/MetricCardContainer.vue

```diff
@@ -47,11 +50,15 @@ import type { PropType } from 'vue'
 import { computed } from 'vue'
 import approxNum from 'approximate-number'
 import { MetricCardSize } from '../constants'
+import type { MetricCardType } from '../enums'
 import type { MetricCardDef, MetricCardDisplayValue } from '../types'
 import { changePolarity, metricChange, defineIcon, calculateChange } from '../utilities'
 import MetricsCard from './display/MetricsCard.vue'
 import MetricCardLoadingSkeleton from './display/MetricCardLoadingSkeleton.vue'
 
+// Only needed for type definitions
```

Comment:
suggestion: update comment
```suggestion
// Import any one of the `@kong/icons` components in order to access the interface (they are all the same); then alias as a generic to provide the icon interface to the prop types.
```